storage/src/context.rs

```diff
@@ -153,10 +153,24 @@ impl ContextApi for TezedgeContext {
         // clients may pass in a prefix with elements containing slashes (expecting us to split)
         // we need to join with '/' and split again
         let prefix = to_key(prefix).split('/').map(|s| s.to_string()).collect();
-        let context_hash_arr: EntryHash = context_hash.as_slice().try_into().expect("EntryHash conversion error");
+        let context_hash_arr: EntryHash = context_hash.as_slice().try_into()?;
         merkle.get_key_values_by_prefix(&context_hash_arr, &prefix)
     }
 
+    fn get_context_tree_by_prefix(&self, context_hash: &ContextHash, prefix: &Option<&str>) -> Result<StringTree, MerkleError> {
+        let merkle = self.merkle.read().expect("lock poisoning");
+        // clients may pass in a prefix with elements containing slashes (expecting us to split)
+        // we need to join with '/' and split again
+        let mut prefix: ContextKey = match prefix {
+            Some(prefix) => prefix.split('/').map(|s| s.to_string()).collect(),
+            None => vec![],
+        };
+        // clients assume that root is at /data
+        prefix.insert(0, "data".to_string());
```

Comment:
@michal-stanek 
I moved this prefix parsing and "data" to the client call, so it could be used to get another key prefixes : test_chain, protocol or other roots in future (if any)


**rpc -> base_services.rs:**
```
    // we assume that root is at "/data"
    let mut key_prefix = vec!["data".to_string()];

    // clients may pass in a prefix (without /data) with elements containing slashes (expecting us to split)
    // we need to join with '/' and split again
    if let Some(prefix) = prefix {
        key_prefix.extend(prefix.split('/').map(|s| s.to_string()));
    };

    match context.get_context_tree_by_prefix(&ctx_hash.unwrap(), &key_prefix) {
```